src/test/java/com/coinbase/exchange/api/orders/OrderTests.java

```diff
@@ -102,10 +102,29 @@ public void getAllOpenOrders() {
     }
 
     @Test
-    public void getFills() {
-        List<Fill> fills = orderService.getAllFills();
+    public void getFillsByProductId() {
+        List<Fill> fills = orderService.getFillsByProductId("BTC-USD", 100);
         assertTrue(fills.size() >= 0);
     }
+
+    @Test
+    public void createMarketOrderBuyThenGetFillByOrderId() {
+        NewMarketOrderSingle marketOrder = createNewMarketOrder("BTC-USD", "buy", new BigDecimal(0.01));
+        Order order = orderService.createOrder(marketOrder);
+
+        assertTrue(order != null); //make sure we created an order
+        String orderId = order.getId();
+        assertTrue(orderId.length() > 0); //ensure we have an actual orderId
+        Order filledOrder = orderService.getOrder(orderId);
+        assertTrue(filledOrder != null); //ensure our order hit the system
+        assertTrue(new BigDecimal(filledOrder.getSize()).compareTo(BigDecimal.ZERO) > 0); //ensure we got a fill
+        log.info("Order opened and filled: " + filledOrder.getSize() + " @ " + filledOrder.getExecuted_value()
```

Comment:
I don't usually see log statements in tests due to them being largely redundant. If something goes wrong, the assertion should fail for the relevant condition.

Reply:
Will remove